src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/AttachedPageBase.java

```diff
@@ -38,4 +39,12 @@ public T open(String redirectUrl) {
 
     return getThis();
   }
+
+  //TODO: Remove it after fandom migration
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)